third_party/dml/ci/pipeline/setup_agent_windows.yml

```diff
@@ -38,7 +38,7 @@ steps:
 
 - powershell: |
     Invoke-Expression '$(miniconda.activateCommand)'
-    pip install six numpy wheel
+    pip install six numpy==1.18.5 wheel
```

Comment:
We should perhaps have a requirements.txt to avoid duplicating this part (it would also be easier for users to install them all). Shrug.